app/controllers/api/v1/groups_controller.rb

```diff
@@ -1,8 +1,9 @@
 class Api::V1::GroupsController < Api::ApiController
   include JsonApiController
 
-  doorkeeper_for :index, :create, :show, scopes: [:public]
```

Comment:
Yes. UserGroup admins can set the visibility of their group (by setting visible_to in this verison or private: true after #418), so if they don't want their groups to be visible they have the option of hiding them.